lldb/include/lldb/Breakpoint/BreakpointLocation.h

```diff
@@ -313,6 +313,8 @@ class BreakpointLocation
 
   void UndoBumpHitCount();
 
+  void SetThreadIDInternal(lldb::tid_t thread_id);
```

Comment:
Could you put a comment in here saying how this is different from SetThreadID?

Reply:
Sure

Reply:
@jimingham how does this look to you?

Reply:
I don't think you need to mention crashing lldb.  Even if we fix that, telling the world about something that's not fully constructed yet is enough of a bad idea, we can leave it at that.
Otherwise this is good.